ow_power_system/src/power_system_node.cpp

```diff
@@ -303,10 +352,13 @@ void PowerSystemNode::parseEoD_Event(const ProgEvent& eod_event, Float64& soc_ms
 void PowerSystemNode::powerCb(double electrical_power)
 {
   // Temperature estimate based on pseudorandom noise and fixed range
-  double voltage_estimate = generateVoltageEstimate();
   double temperature_estimate = generateTemperatureEstimate();
-  injectFaults(electrical_power, voltage_estimate, temperature_estimate);
-  auto current_data = composePrognoserData(electrical_power, voltage_estimate, temperature_estimate);
+  double voltage_estimate = generateVoltageEstimate();
+  double adjusted_wattage = std::max(electrical_power, BASELINE_WATTAGE);
```

Comment:
If the baseline power accounts for always-on subsystems (computer, heater, ect.) shouldn't it be added to the total electrical power?

Reply:
I was interpreting it as "minimum power" but I think your take is right.  Changed, and included a comment.